### PR TITLE
Created notification component

### LINKED ISSUE #122: Notification component

### PR DESCRIPTION
**Pull request by shwetalsoni**
### Fixes #122 

Screenshot (332)

### CODE REVIEW

File: frontend/src/components/Notification.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Avatar from "./Avatar.jsx";
+
+//images
+import avatar_image from "../images/placeholder-images/avatar.png";
+
+class Notification extends Component{
+
+    render(){
+        return(
+            <div className="notification">
```
Comment (salil-naik) [inline frontend/src/components/Notification.jsx:11]: className="review-notification" is more acurate

File: frontend/src/components/Notification.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                <div className="notification__profile-wrap">
+                    <Avatar avatarSrc={avatar_image}/>
+                    <h3 className="notification__username">@Shwetal </h3>
+                    <p className="notification__article">added this article</p>
+                </div>
+                <div className="notification__review-wrap">
+                    <a href="#" className="button__xsmall button__xsmall--hollow">Review</a>
```
Comment (salil-naik) [inline frontend/src/components/Notification.jsx:18]: In the design, you have used the small variant of the button. Was there any reason you had to make xsmall variant? 🤔

**Review by salil-naik — CHANGES_REQUESTED** [review]
Hi. 

- [ ] Keep 30px padding on left and right. I have updated the design in Figma for your reference. 

- [ ] The text on the right is not aligned properly. (vertically)

- [ ] Resolve conflicts if any.

File: frontend/src/components/Notification.jsx (hunk 2 above)
Comment (shwetalsoni) [reply frontend/src/components/Notification.jsx:18]: @salil-naik 
as per figma design i thought that the buttons existing in _button.scss won't fit here. they were too big for notification component.....So, had to create that xsmall variant

File: frontend/src/components/Notification.jsx (hunk 1 above)
Comment (shwetalsoni) [reply frontend/src/components/Notification.jsx:11]: *The text on the right is not aligned properly. (vertically)*
By this you mean button on right ?

File: frontend/src/components/Notification.jsx (hunk 2 above)
Comment (salil-naik) [reply frontend/src/components/Notification.jsx:18]: Use `button__small button__small--hollow`

File: frontend/src/App.js [hunk 3]
```diff
@@ -10,23 +10,28 @@ import Header from "./components/Header.jsx";
 import Notification from"./components/Notification.jsx";
 
 //images
```
Comment (salil-naik) [inline frontend/src/App.js:12]: Remove this line

File: frontend/src/App.js [hunk 4]
```diff
@@ -10,23 +10,28 @@ import Header from "./components/Header.jsx";
...
-import avatar_image from "./images/placeholder-images/avatar.png";
+// import avatar_image from "./images/placeholder-images/avatar.png";
```
Comment (salil-naik) [inline frontend/src/App.js:13]: Remove this line.

**Review by salil-naik — CHANGES_REQUESTED** [review]

File: frontend/src/components/Notification.jsx (hunk 1 above)
Comment (salil-naik) [reply frontend/src/components/Notification.jsx]: I'm talking about added this article and avatar.

File: frontend/src/components/Notification.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import avatar_image from "../images/placeholder-images/avatar.png";
+
+class Notification extends Component{
+
+    render(){
+        return(
+            <div className="review-notification">
+                <div className="review-notification__profile-wrap">
+                    <Avatar avatarSrc={avatar_image}/>
+                    <h3 className="review-notification__username">@Shwetal </h3>
```
Comment (salil-naik) [inline frontend/src/components/Notification.jsx:14]: Replace h3 with p

File: frontend/src/App.js [hunk 6]
```diff
@@ -18,14 +18,15 @@ import CodeOfConduct from "./pages/CodeOfConduct.jsx";
...
+
 // images
 import avatar_image from "./images/placeholder-images/avatar.png";
 
 function App() {
   return (
     <Router>
       <Header isSignedIn={false} avatarSrc={avatar_image} />
-
+     <br/>
```
Comment (salil-naik) [inline frontend/src/App.js:29]: Why is this added?

File: frontend/src/css/components/_notification.scss [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        padding-top: 5px;
+        color: #2F80ED;
+    }
+
+    &__article{
+        margin-left: 5px;
+        padding-top: 5px;
+    }
+
+    .button__small--hollow{
```
Comment (salil-naik) [inline frontend/src/css/components/_notification.scss:30]: I don't think you need to rewrite the button CSS. Remove this.

**Review by salil-naik — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 6 above)
Comment (shwetalsoni) [reply frontend/src/App.js:29]: I haven't. Should i remove this ?

File: frontend/src/css/components/_notification.scss (hunk 7 above)
Comment (shwetalsoni) [reply frontend/src/css/components/_notification.scss:30]: As per the figma design , it has a purple border. So i had to add that

File: frontend/src/App.js (hunk 6 above)
Comment (salil-naik) [reply frontend/src/App.js:29]: Yes. Remove.

File: frontend/src/css/components/_notification.scss (hunk 7 above)
Comment (salil-naik) [reply frontend/src/css/components/_notification.scss:30]: Yes. Remove it. I just noticed the buttons are not the same in design and code. I'll fix the Figma buttons later on.
